ledger-api/grpc-definitions/com/daml/ledger/api/v1/version_service.proto

```diff
@@ -22,6 +22,11 @@ message GetLedgerApiVersionRequest {
   // Must be a valid LedgerString (as described in ``value.proto``).
   // Required
   string ledger_id = 1;
+
+  // Whether synchronous API-level errors include the error details as part of the error message string,
+  // as explained in `error.proto`.
+  // Optional
+  bool error_details_in_message = 1001;
```

Comment:
@andreaslochbihler-da : the changes wrt `error_details_in_message` are cluttering the API surface. I'm wondering whether these wouldn't be better communicated via an extra header being set. That would also allow adding that parameter transparently by middle-ware.

I'd suggest revisiting this choice, and potentially factoring it out of this PR.

Reply:
Rewritten in error.proto and added a TODO